### PR TITLE
Added Splash Screen

### PR DESCRIPTION
**Pull request by ahmadrns**
Solve issue mentioned in https://github.com/Pylons-tech/pylons/issues/1342

### CODE REVIEW

**Comment by kjawadDeveloper2** [comment]
@ahmadrns  images size is too high Get svg from @StefanB90

File: wallet/lib/pages/routing_page/splash_screen.dart [hunk 1]
```diff
@@ -12,31 +14,50 @@ import 'package:pylons_wallet/services/data_stores/local_data_store.dart';
...
+    timer = Timer.periodic(
+      const Duration(seconds: 1),
+      (timer) {
+        if (index > 4) {
+          checkAppLatestOrNot().then((value) {
+            userInfoProvider.initIPC();
+          });
+          return;
+        }
+        setState(() {
```
Comment (kjawadDeveloper2) [inline wallet/lib/pages/routing_page/splash_screen.dart:56]: Why not value notifier here?

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

**Comment by ahmadrns** [comment]
> @ahmadrns images size is too high Get svg from @StefanB90

sure

**Comment by ahmadrns** [comment]
@StefanB90  @xinyapylons Kindly share `svg` for these images as the size of these images is too high

File: wallet/lib/pages/routing_page/splash_screen.dart (hunk 1 above)
Comment (ahmadrns) [reply wallet/lib/pages/routing_page/splash_screen.dart:56]: sure, will refactor it

**Review by kjawadDeveloper — APPROVED** [review]

**Review by Amean — APPROVED** [review]

**Review by kjawadDeveloper2 — APPROVED** [review]

**Pull request merged**